codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerAwsJson.kt

```diff
@@ -36,8 +35,6 @@ class ServerAwsJsonFactory(private val version: AwsJsonVersion) : ProtocolGenera
     override fun buildProtocolGenerator(codegenContext: CodegenContext): ServerHttpBoundProtocolGenerator =
         ServerHttpBoundProtocolGenerator(codegenContext, protocol(codegenContext))
 
-    override fun transformModel(model: Model): Model = model
```

Comment:
I used this yesterday to augment the S3 model with some extra metadata—I think we should keep these, especially since the default implementation means that there isn't any cost to having it

Reply:
You used the `transformModel` from the decorators, not this one. I accidentally removed the invocation of the former, but I didn't mean to, that's why CI failed. I've now fixed that.